test/random.hpp

```diff
@@ -40,12 +40,14 @@
 /// But we want this functionality to be independent, so changes in tests won't affect the driver
 /// and vice versa. This independency could be important, because, for example, the driver
 /// implements its own cache of verification data and change or GET_RAND() would break it.
+namespace test {
```

Comment:
[Q] Do we really we need this? GET_RAND is not included into the library and thus does not litter the global namespace.